.github/workflows/test.yml

```
@@ -13,92 +13,98 @@ jobs:
-
-    - name: 🔍 Lint code
-      uses: golangci/golangci-lint-action@v4
-      with:
-        version: latest
-        working-directory: api
-        args: --timeout=5m
+      - name: 📥 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🐹 Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.8)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)